
string  sensor_state
bool    slip_status
float64 slip_speed
float64 slip_angle
float64 estimated_force
std_msgs/Float64MultiArray estimated_location
